Explorer/Assets/DCL/Multiplayer/Profiles/Entities/RemoteEntities.cs

```diff
@@ -109,16 +109,15 @@ bool ContainsInRoom(IRoom room)
             return ContainsInRoom(roomHub.IslandRoom()) || ContainsInRoom(roomHub.SceneRoom());
         }
 
-        private void TryCreateOrUpdateRemoteEntity(in RemoteProfile profile, World world)
+        public Entity TryCreateOrUpdateRemoteEntity(in RemoteProfile profile, World world)
```

Comment:
Why to make it public and change contract here?

Reply:
Because we need to Create RemoteEntity from the Debug menu now as well, and that is why requires direct call of this method